gateway-service-impl/src/test/java/org/hypertrace/gateway/service/entity/EntityServiceTest.java

```diff
@@ -275,6 +266,27 @@ public void testGetEntitiesOnlySelectFromMultipleSources() {
             .addSelection(getExpressionFor("API.apiName", "API Name"))
             .addSelection(getExpressionFor("API.httpMethod", "API Http method"))
             .build();
+    QueryRequest expectedQueryRequest = QueryRequest.newBuilder()
+        .setFilter(
+            QueryServiceRequestAndResponseUtils.createQsDefaultRequestFilter("API.startTime",
```

Comment:
This filter always has a time range filter in it, there by verified that the query request has time range filter.